features/people.feature

```diff
@@ -13,11 +13,8 @@ Scenario: Viewing the person page for a person
 
 Scenario: Announcements by that person over their career are shown
   Given I am an editor
-  And "Don Deputy" is the "Deputy Prime Minister" for the "Cabinet Office"
   And "Harriet Home" is the "Home Secretary" for the "Cabinet Office"
```

Comment:
Is this step necessary anymore?

Reply:
This creates the actual person record for "Harriet".

Reply:
Ah yes. One more thing - could you remove the 'over the career' bit from the scenario? It doesn't make sense anymore.

Reply:
Done!